datafusion/core/tests/sql/parquet.rs

```diff
@@ -173,58 +173,6 @@ async fn parquet_list_columns() {
     assert_eq!(result.value(3), "xyz");
 }
 
-#[tokio::test]
```

Comment:
I moved this to its own module `parquet_format` as it was getting somewhat complicated